pkg/operator/ceph/cluster/osd/volumes.go

```diff
@@ -176,16 +177,16 @@ func getUdevVolume() (v1.Volume, v1.VolumeMount) {
 	return volume, volumeMounts
 }
 
-func getEncryptionVolume(pvcName string) (v1.Volume, v1.VolumeMount) {
+func getEncryptionVolume(osdProps osdProperties, isKMS bool) (v1.Volume, v1.VolumeMount) {
```

Comment:
can't you infer the `isKMS` from the osd properties which has the kms settings?
```suggestion
func getEncryptionVolume(osdProps osdProperties) (v1.Volume, v1.VolumeMount) {
```

Reply:
Yes, can do.